libs/blocks/preflight/panels/general.js

```diff
@@ -64,6 +70,8 @@ async function setContent() {
     page: { items: [{ url: new URL(window.location.href), edit: null, preview: 'Fetching', live: 'Fetching' }] },
     fragments: { items: findLinks('main .fragment, a[data-modal-path], [data-path]') },
     links: { items: findLinks('main a[href^="/"') },
+    svgs: { items: findLinks('img[src$=".svg"') },
```

Comment:
I'd guess it's certain that these checks run after everything is decorated? Otherwise, would we need to check for anchor tags referencing SVG resources, as that's how they're added via authoring?

Reply:
Yes, preflight can only be launched from the tools dropdown on a `.page` or `.live` page.